### DISCUSSION TITLE
looking good, until I looked deeper

### DISCUSSION THREAD

**Discussion by weisk** [said7388/developer-portfolio#27] [Ideas]
Yeah , so I was almost ready to hire you, but then I was reading the snippet that programatically describes your coder properties.
I saw your `hireable` function, but then i thought.... why is this even a function if the boolean expression uses its own hardcoded values only, so why not just put the boolean expression outside and execute it only once, so hireable will not bea function but a boolean. Even more, why bother with logical operations when you know it's going to be `true` always, just hardcode true aswell.

Ok now joking aside.... hehehe
It's looks cool. Here's a suggestion:
Make your `hireable` function , a function that accepts inputs. Example:
```
(requirements, offer) => {
...
``` 

and inside, you do some stuff with that.
Like what if my offer says: `{salary:  1M dollars/year}`, and my requirements contain: `{daysPerWeek: 6, minWeekHours: 60, yearlyVacationDays: 1}`

think about that ;)

**Comment by said7388** [comment]
Hey @weisk , 
Thank you for your observation and giving feedback. The code snippet in the header section is just for design purposes. I used a function for hireable properties instead of a boolean expression to align the code snippets' height with the left-side content. If I used a boolean expression its height would not be the same as left side content 😊
